docs/source/operations/fabric_verify.md

```diff
@@ -78,66 +78,162 @@ The flow chart shows the Fabric Deployment process. To verify the steps of deplo
 #### Final network validy check
 For final checking of the validity of the fabric network.
 
-1. Create a cli pod for any organization. (Now Peer CLI can be enabled from network.yaml itself. Check the [sample network.yaml](./../operations/fabric_networkyaml.md) for reference)
+- Create a cli pod for any organization. (Now Peer CLI can be enabled from network.yaml itself. Check the [sample network.yaml](./../operations/fabric_networkyaml.md) for reference)
+
+  Use this sample template.
+  ```yaml
+    metadata:
+      namespace: <ORG_NAME>-net
+    images:
+      fabrictools: hyperledger/fabric-tools:2.0
+      alpineutils: index.docker.io/hyperledgerlabs/alpine-utils:1.0
+    storage:
+      class: <ORG_NAME>sc
+      size: 256Mi
+    vault:
+      role: ault-role
+      address: <VAULT_ADDR>
+      authpath: <ORG_NAME>-net-auth
+      adminsecretprefix: secret/crypto/peerOrganizations/<ORG_NAME>-net/users/admin
+      orderersecretprefix: secret/crypto/peerOrganizations/<ORG_NAME>-net/orderer
+      serviceaccountname: vault-auth
+      imagesecretname: regcred
+      tls: false
+    peer:
+      name: <PEER_NAME>
+      localmspid: <ORG_NAME>MSP
+      tlsstatus: true
+      address: <PEER_NAME>.<ORG_NAME>-net.<EXTERNAL_URL_SUFFIX>:8443
+    orderer:
+      address: <ORDERER_NAME>
+  ```
+
+- To install the cli
+  ```
+  helm install -f cli.yaml /blockchain-automation-framework/platforms/hyperledger-fabric/charts/fabric_cli/ -n <CLI_NAME>
+  ```
+
+- Get the cli pod
+  ```
+  export ORG1_NS=<ORG_NAME>-net
+  export CLI=$(kubectl get po -n ${ORG1_NS} | grep "cli" | awk '{print $1}')
+  ```
+
+- Copy the cli pod name from the output list and enter the cli using.
+  ```
+  kubectl exec -it $CLI -n <ORG_NAME>-net -- bash
+  ```
+
+- To see which chaincodes are installed
+  ```
+  peer chaincode list --installed (after exec into the cli)
+  ```
+
+- Check if the chaincode is instantiated or not
+  ```
+  peer chaincode list --instantiated -C allchannel (after exec into the cli)
+  ```
+
+- Execute a transaction
+  For init:
+  ```
+  peer chaincode invoke -o <orderer url> --tls true --cafile <path of orderer tls cert> -C <channel name> -n <chaincode name> -c '{"Args":[<CHAINCODE_INSTANTIATION_ARGUMENT>]}' (after exec into the cli)
+  ```
+
+Upon successful invocation, should display a `status 200` msg.
+
+
+---  
+
+
+## Hyperledger Indy Checks
+
+The flow chart shows the Indy Deployment process. To verify the steps of deployment, follow the Verification Table 'I', to troubleshoot the general errors.
+
+![](./../_static/indy_flowchart.png)
 
-Use this sample template.
-```yaml
+----
+### Indy Troubleshooting
+### Table 'I'

```

Comment:
Flowchart steps are wrong. A flow chart Process box should give an instruction like "Do x", "Check y", not just "Domain Genesis". Please check how it is worked in Fabric (difference between pods and jobs).

Reply:
done